frontend/public/components/catalog-item-details.jsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+import {Link} from 'react-router-dom';
+import {Modal} from 'patternfly-react/dist/esm/components/Modal';
+import {CatalogItemHeader} from 'patternfly-react-extensions/dist/esm/components/CatalogItemHeader';
+import {PropertiesSidePanel, PropertyItem} from 'patternfly-react-extensions/dist/esm/components/PropertiesSidePanel';
+
+import {normalizeIconClass} from './catalog-item-icon';
+import {ClusterServicePlanModel} from '../models';
+import {k8sGet} from '../module/k8s';
+import {Timestamp} from './utils';
+
+export class CatalogTileDetails extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      plans: [],
+    };
+  }
+
+  componentDidMount() {
+    const { obj } = this.props.item;
+    this.getPlans(obj);
```

Comment:
We should only call this for service classes, not other items like source to image builders